src/gis/proj.md

```diff
@@ -1,13 +1,55 @@
+
+If you want to learn more about this topic, we recommend to go through the <QGISHelp ver="3.22" link="gentle_gis_introduction/coordinate_reference_systems.html" text="QGIS online documentation" /> or [PROJ documentation](https://proj.org/operations/index.html).
 
-You can read more about Map Projections and Coordinate Reference Systems in this [extended article](./projections/index.md).
+Projections issues are one of the common causes of misplacement of your field data as mentioned in our blog [Why are my survey points shifted?](https://www.lutraconsulting.co.uk/blog/2021/04/21/projections-field/).
 
-## Using custom PROJ
+## Custom projections in Mergin Maps Input
+If you had to install grid shift files in QGIS in order to display your layers correctly, it is also necessary to provide the same grid shift files to <MobileAppName />.
 
-When <MobileAppName /> is launched, it searches for `<project folder>/proj` for custom PROJ datum shifts files in all available projects on the disk. These shift files can be then used in all projects for custom PROJ datum shift.
+<MobileAppName /> supports grid shift files formats:
+- *National Transformation version 2* (<NoSpellcheck id=".gsb" />) <Badge text="Since Input 0.8.0" type="info"/>
+- *Geodetic TIFF grids* (.tiff) <Badge text="Since Input 1.7.0" type="info"/>
 
-If you want to use your custom PROJ datum shifts, copy them to the folder `<project folder>/proj` to correctly render your QGIS project on <MobileAppName />. Note that one the project is firstly downloaded to <MobileAppName />, you have to restart <MobileAppName /> to load your custom datum shifts.
+When the app starts, it searches for `proj` folders in all available projects on the disk. The grid shift files found in these folders can be then used in all projects.
 
-We recommend to use Mergin service to transfer the project with the required proj resources to your mobile device.
+To add grid shift files to <MobileAppName />:
+1. Create a folder called `proj` in your project folder
+   ![proj folder](./proj-folder.png)
+2. Copy the grid shift file to the `proj` folder
+   ![shift file](./proj-folder-shift-file.png)
+   
+   :::tip
+   Grid shift files installed in QGIS on your computer can be found in these locations:
+   - Windows users: `C:\Users\USER\AppData\Roaming\QGIS\QGIS3\profiles\default\proj` (USER is your username)
+   - macOS users: `~/Library/Application\Support/QGIS/QGIS3/profiles/default/proj`
+   - Linux users: `~/.local/share/QGIS/QGIS3/profiles/default/proj`
+   :::
+   
+3. Synchronise the project using <QGISPluginName />
+4. Download the project with the grid shift files to <MobileAppName />. 
+   When the project is downloaded to the app for the first time, you will get a **PROJ Error** that will prompt you to **restart the app** to load the grid shift files. The error message should not appear after the restart of the app.
+![input shift grid](./input-custom-prj.png)
```

Comment:
In the upcoming release of Mergin plugin, this should be simplified - the plugin should check for any grids used by the project automatically and offer an automatic fix if the grid file is missing (will copy the grid(s) to `proj` sub-dir). Also if a new project is created in the wizard, grids will get packaged automatically. So maybe the points 1+2 can be kept as a manual procedure for older plugin version users (or a fallback for those who want to do things manually)

see https://github.com/MerginMaps/qgis-mergin-plugin/pull/455